src/routes/ticketcontrol/+page.svelte

```diff
@@ -87,7 +166,7 @@
         background-color: black;
         width: 100%;
     }
-    .container{
+    container{
```

Comment:
pls use tailwind as we use it everywhere else tailwind is real easy. this would be class="bg-black w-full" for example